docs/integrations/sources/shopify.md

```diff
@@ -3,14 +3,14 @@ description: >-
   Shopify is a proprietary e-commerce platform for online stores and retail point-of-sale systems.
 ---
 
-::: note
+# Shopify
```

Comment:
:information_source: The header must always be the top entry, so docusaurus can correctly parse it out. We should never have content before the first header.

Also it's `:::note` not `::: note` (no space allowed!)